### PR TITLE
Converted read more alert to joomla-alert

### PR DESCRIPTION
**Pull request by JackKellyUK**
Pull Request for Issue #38991.

### Summary of Changes

Converted read more alert to joomla-alert (not sure if this should be a warning instead of error).

### Testing Instructions

1. Create a new Article
2. Insert a read more
3. Attempt to insert another read more
4. Verify message string

### Actual result BEFORE applying this Pull Request

Message displayed in js alert with broken ellipses

### Expected result AFTER applying this Pull Request

Message displayed within Joomla's message system correctly

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: build/media_source/com_content/js/admin-article-readmore.es6.js [hunk 1]
```diff
@@ -20,8 +20,9 @@
     } else if (content && !content.match(/<hr\s+id=("|')system-readmore("|')\s*\/*>/i)) {
       Joomla.editors.instances[editor].replaceSelection('<hr id="system-readmore">');
     } else {
-      // @todo replace with joomla-alert
-      alert(options.exists);
+      Joomla.renderMessages({
+        error: [options.exists]
```
Comment (richard67) [inline build/media_source/com_content/js/admin-article-readmore.es6.js]:
```suggestion
        error: [options.exists],
```
Fix javascript code style error reported by drone here: https://ci.joomla.org/joomla/joomla-cms/60552/1/26

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on c5c2f9d3a518c8c7b6d4c68cb7aacbae520f1e67<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39573">issues.joomla.org/tracker/joomla-cms/39573</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39573">issues.joomla.org/tracker/joomla-cms/39573</a>.</sub>

**Comment by roland-d** [comment]
Thank you

**Pull request merged**